docs/pages/includes/device-trust/prereqs.mdx

```diff
@@ -12,3 +12,6 @@
   - `tsh` v15.0.0 or newer. [Install tsh for Linux](../../installation.mdx#linux).
 - To authenticate a Web UI session you need [Teleport Connect](
   ../../connect-your-client/teleport-connect.mdx#installation--upgrade)
+- Correct end-user IP propagation to your Teleport deployment: X-Forwarded-For
```

Comment:
can we add https://goteleport.com/docs/reference/config/#reference-configurations as reference?

```
    # Enables the Proxy service to take client source IPs from the
    # "X-Forwarded-For" headers for web APIs.
    #
    # In order to prevent IP spoofing, it is important to ensure that the Proxy
    # service is placed behind a layer 7 load balancer or reverse proxy that
    # sets client IPs in the "X-Forwarded-For" headers. Requests with more than
    # one IPs will be rejected.
    trust_x_forwarded_for: false
```

We really need a doc to guide people ensuring the IP addresses are correct and how to configure them. That would be awesome using debug app access to validate them. 

cc @ptgott

Reply:
Thanks